File: springbootjparest/src/main/java/at/eder/springbootjparest/models/Company.java
package at.eder.springbootjparest.models;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Data
@Entity
@NoArgsConstructor
@SuperBuilder
public class Company {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String name;
    private String zipTown;
    private String street;
    private String phone;
    private String eMail;
    private String replyTo;
    private String comments;
}

File: WTMobileApp/app/src/main/java/at/eder/wirtschaftstagmobileapp/models/Company.kt
package at.eder.wirtschaftstagmobileapp.models

data class Company(
    var id: Long?,
    var name: String?,
    var zipTown: String?,
    var street: String?,
    var phone: String?,
    var email: String?,
    var replyTo: String?,
    var comments: String?
) {
    override fun toString(): String = "$id - $name: $zipTown"
}